extensions/binding/pom.xml

```diff
@@ -34,6 +34,8 @@
     <module>org.eclipse.smarthome.binding.lirc</module>
     <module>org.eclipse.smarthome.binding.mqtt</module>
     <module>org.eclipse.smarthome.binding.mqtt.test</module>
+    <module>org.eclipse.smarthome.binding.onewire</module>
```

Comment:
Please also add the binding to:
- https://github.com/eclipse/smarthome/blob/master/features/karaf/esh-ext/src/main/feature/feature.xml
- https://github.com/eclipse/smarthome/blob/master/features/org.eclipse.smarthome.feature.runtime.binding/feature.xml
- https://github.com/eclipse/smarthome/blob/master/docs/_includes/documentation-menu.html